ayushma/models/users.py

```diff
@@ -5,20 +5,38 @@
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 
+def reverse_choices(choices):
```

Comment:
We are using enums for choices everywhere in Ayushma. Can you convert this to use enums? Define them in the enums.py file.